cocos/core/geometry/spline.ts

```diff
@@ -76,10 +76,10 @@ export enum SplineMode {
 
 const SPLINE_WHOLE_INDEX = 0xffffffff;
 
-const _v0 = v3();
-const _v1 = v3();
-const _v2 = v3();
-const _v3 = v3();
+const _v0 = new Vec3();
+const _v1 = new Vec3();
+const _v2 = new Vec3();
+const _v3 = new Vec3();
```

Comment:
Why change it?

Reply:
Revert them. 
I was thinking `v3()` is shorter than `new Vec3()` which could save some code size.